tests/commands/device/supported.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import * as _ from 'lodash';
+import { runCommand } from '../../helpers';
+
+const HELP = `
+Usage: devices supported
+
+Use this command to get the list of all supported devices.
+
+Examples:
+
+\t$ balena devices supported
+`;
+
+const cleanOutput = (output: string[] | string) => {
```

Comment:
Is this a copy-and-paste of the same function in `tests/helpers.ts`? Maybe it can be imported instead?
Btw, I think it's just a personal preference of mine, but I'd declare the function with the `function` keyword instead of `const`:
```
function cleanOutput(output: string[] | string) {
```

Reply:
Ah, actually looks like I'd added the function to `tests/helpers.ts` myself in this PR, but then forgot to import it here, doh :)  Will update.

Regarding the style, yeah I share your preference!  I think I had originally just copied this from `tests/commands/help.spec.ts` and so had not modified the style.

I can also change the other tests (e.g. help.spec.ts) to import this function while I'm here.